app/bin/quasar-build

```diff
@@ -42,14 +42,17 @@ if (argv.help) {
     $ quasar build -m ios -- some params --and options --here
 
   Options
-    --mode, -m      App mode [spa|ssr|pwa|cordova|electron] (default: spa)
+    --mode, -m      App mode [spa|ssr|pwa|cordova|electron|tauri] (default: spa)
```

Comment:
"Tauri is in alpha stage, anything can change"

Reply:
added at line 53

Reply:
maybe quasar dev should point that too

Reply:
I'm on it.

Reply:
done